### ISSUE TITLE
Does not work on windows

### ISSUE THREAD

**Issue by xvik** [koral--/jacoco-gradle-testkit-plugin#1]
Plugin prepares bad agent path on windows and testkit fails to run.

Fix: https://github.com/koral--/jacoco-gradle-testkit-plugin/blob/master/src/main/kotlin/pl/droidsonroids/gradle/jacoco/testkit/GenerateJaCoCoTestKitProperties.kt#L20

```
val jacocoRuntimePath = jacocoRuntimeConfiguration.asPath.replace('\\', '/')
```

**Comment by koral--** [comment]
Thanks, I'll change it ASAP.

**Comment by koral--** [comment]
Fixed in version `1.0.1`.
I've also added CI on Windows machine using [Appveyor](https://ci.appveyor.com/project/koral--/jacoco-gradle-testkit-plugin/build/master%205/job/r7x0k2ygys8gitw1)

**Comment by xvik** [comment]
Thank you!

**Comment by xvik** [comment]
I was just barely checked plugin before but now, when I started to use it in the real project, it appears that fix was incomplete.

The problem comes from the default value: https://github.com/koral--/jacoco-gradle-testkit-plugin/blob/master/src/main/kotlin/pl/droidsonroids/gradle/jacoco/testkit/GenerateJaCoCoTestKitProperties.kt#L17

```kt
var destinationFile = "${project.buildDir}/jacoco/test.exec"
```

project.buildDir returns windows path so overall path looks like: `C:\dev\project\build/jacoco/test.exec`
No errors appear - it just did not write testkit coverage. 

The fix is the same as before (`.replace("\\", '/')`). For now, I have to do it in build directly:

```
generateJacocoTestKitProperties.destinationFile = file('build/jacoco/test.exec').absolutePath.replace("\\", '/')
```

**Comment by koral--** [comment]
Thanks, I'll try to fix it ASAP.

**Comment by hakanai** [comment]
Taking a quick look at presumably the previous attempt at a fix:

https://github.com/koral--/jacoco-gradle-testkit-plugin/blob/master/src/main/kotlin/pl/droidsonroids/gradle/jacoco/testkit/GenerateJaCoCoTestKitProperties.kt#L29

As far as I can tell, the argument to `-javaagent` is supposed to be a file path, so backslashes seem to be correct. However, spaces in the path might break things.

The current behaviour of actually writing the file:

```
        outputFile.writeText("org.gradle.jvmargs:-javaagent:$jacocoRuntimePath=destfile=$destinationFile")
```

The fix might be as simple as:

```
        outputFile.writeText("org.gradle.jvmargs:\"-javaagent:$jacocoRuntimePath=destfile=$destinationFile\"")
```

with the conversion from `\` to `/` completely unnecessary for both parameters, as both seem to want file paths.

**Comment by koral--** [comment]
Fix released in version 1.0.4, thanks.
